firestore/integration_test_internal/src/android/snapshot_metadata_android_test.cc

```diff
@@ -2,18 +2,24 @@
 
 #include "firestore/src/include/firebase/firestore/snapshot_metadata.h"
 #include "firestore/src/jni/env.h"
-#include "firestore/src/tests/firestore_integration_test.h"
-#include "testing/base/public/gmock.h"
+#include "firestore_integration_test.h"
+#include "android/firestore_integration_test_android.h"
+#include "gmock/gmock.h"
 #include "gtest/gtest.h"
+#include "firebase_test_framework.h"
```

Comment:
nit: and alphabetize here.

Reply:
firebase_test_framework and app_framework are additions (so that we can run their tests in our own test framework). All other includes are inherited. I'll hold off on alphabetizing these files until we can set up code formatting and alphabetize all.